tex/alg-NT/ramification.tex

```diff
@@ -190,9 +215,13 @@ \section{The magic of Galois extensions}
 	Since $\NK(x)$ is an integer and divisible by $\kp_1$,
 	we should have that $\NK(x)$ is divisible by $p$.
 	Thus it should be divisible by $\kp_2$ as well.
-	But by the way we selected $x$, we have $x \notin \sigma\inv\kp_2$ for every $\sigma \in G$!
-	So $\sigma(x) \notin \kp_2$ for any $\sigma$, which is a contradiction.
+	Thus, for some $\sigma \in \Gal(K/\QQ)$, $\sigma(x)$ is divisible by
+	$\kp_2$, equivalently, $x$ is divisible by $\sigma\inv\kp_2$.
+	But by the way we selected $x$, we have within the factors of $p$, $x$ is
+	divisible by only $\kp_1$!
+	So $\sigma\inv\kp_2 = \kp_1$, and we're done.
```

Comment:
We don't need prove by contradiction. Makes the proof a bit cleaner I think. (also to see which assumption is needed to do what, for example you don't need that 𝔭₁ cannot be carried to 𝔭₂ to pick x)